src/prop-types/current-user.js

```diff
@@ -0,0 +1,45 @@
+/*
+ This file is a part of libertysoil.org website
+ Copyright (C) 2016  Loki Education (Social Enterprise)
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import { PropTypes } from 'react';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+
+import { uuid4 } from './common';
+import { Hashtag } from './hashtags';
+import { School } from './schools';
+import { Geotag } from './geotags';
+import { User } from './users';
+
+
+export const CurrentUser = ImmutablePropTypes.contains({
```

Comment:
It's already implemented: `CurrentUser` in `src/prop-types/users`. What's the reason of this alternative?